### PR TITLE
List Interface - int lastIndexOf(Object o)

### LINKED ISSUE #16: Implement : List Interface - int lastIndexOf(Object o)

### PR DESCRIPTION
**Pull request by vatsalkesarwani12**
#16 Implement : List Interface - int lastIndexOf(Object o)

Fixes #16 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- Java
- 

#### Files Added:
- ListLastIndexOf
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

Keep Contributing. Keep Learning!

### CODE REVIEW

File: Collection Interfaces/List Interface/ListLastIndexOf/LastIndexOf ( Object o ).java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+5
+1 2 3 2 5
+6
+
+Output: 
+
+Initial List : [1, 2, 3, 4, 5]
+LastIndex of 2 in list : -1
+
+*/
```
Comment (jyotijauhari) [inline Collection Interfaces/List Interface/ListLastIndexOf/LastIndexOf ( Object o ).java]: blank line at the end

File: Collection Interfaces/List Interface/ListLastIndexOf/LastIndexOf ( Object o ).java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+Sample TestCase:
+
+Input: 
+5
+1 2 3 2 5
+2
+
+Output: 
+
+Initial List : [1, 2, 3, 4, 5]
```
Comment (jyotijauhari) [inline Collection Interfaces/List Interface/ListLastIndexOf/LastIndexOf ( Object o ).java:50]: Initial list written is wrong . do the necessary changes

File: Collection Interfaces/List Interface/ListLastIndexOf/LastIndexOf ( Object o ).java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+Last Index of 2 in list : 3
+
+Input: 
+5
+1 2 3 2 5
+6
+
+Output: 
+
+Initial List : [1, 2, 3, 4, 5]
```
Comment (jyotijauhari) [inline Collection Interfaces/List Interface/ListLastIndexOf/LastIndexOf ( Object o ).java:60]: changes in Initial List : [1, 2, 3, 4, 5]

File: Collection Interfaces/List Interface/ListLastIndexOf/LastIndexOf ( Object o ).java [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+LastIndex of 2 in list : -1
```
Comment (jyotijauhari) [inline Collection Interfaces/List Interface/ListLastIndexOf/LastIndexOf ( Object o ).java]: changes (we r finding 6 not 2)

File: Collection Interfaces/List Interface/ListLastIndexOf/LastIndexOf ( Object o ).java (hunk 1 above)
Comment (jyotijauhari) [reply Collection Interfaces/List Interface/ListLastIndexOf/LastIndexOf ( Object o ).java]: @plazzy99

**Review by jyotijauhari — CHANGES_REQUESTED** [review]
do the changes asked.

File: Collection Interfaces/List Interface/ListLastIndexOf/LastIndexOf ( Object o ).java (hunk 1 above)
Comment (vatsalkesarwani12) [reply Collection Interfaces/List Interface/ListLastIndexOf/LastIndexOf ( Object o ).java]: done

File: Collection Interfaces/List Interface/ListLastIndexOf/LastIndexOf ( Object o ).java (hunk 4 above)
Comment (vatsalkesarwani12) [reply Collection Interfaces/List Interface/ListLastIndexOf/LastIndexOf ( Object o ).java]: done

**Comment by vatsalkesarwani12** [comment]
@JyotiJauhari changes done

**Review by Dhruv-194 — APPROVED** [review]
Changes have been done as per the request

**Pull request merged**